components/user/UserPage.js

```diff
@@ -9,6 +9,24 @@ import UserEvents from "./UserEvents";
 
 export default function UserPage({ data, BASE_URL }) {
   const [userData, setUserData] = useState(data);
+
+  const userProfileData = useMemo(
+    () => ({
+      name: userData.name,
+      username: userData.username,
+      socials: userData.socials,
+      bio: userData.bio,
+      tags: userData.tags,
+    }),
+    [
+      userData.bio,
+      userData.name,
+      userData.socials,
+      userData.tags,
+      userData.username,
+    ]
+  );
```

Comment:
But none of the user data changes, is this required?

Reply:
It is not because of user data. We are just extracting only the required user information for userProfile component.
Using memo to avoid assignment of new address on every re-render. 

It is used to remove reviews, testimonials ... all the tabs data.

Reply:
thank you for expelling @psavan1655 👍

Reply:
Welcome @eddiejaoude .
Btw when are you planning to implement the user add by form feature? As i want to join with you during that.

Reply:
Yes I started on that 👍 I should push what I have so far to a draft PR. I will try to do that this weekend

Reply:
Can you refer me other open source react related projects? with some intermediate-experienced level codebsae?
I would like to contribute more.